tests/kafkatest/tests/connect/connect_distributed_test.py

```diff
@@ -929,7 +940,12 @@ def test_exactly_once_source(self, clean, connect_protocol, metadata_quorum, use
         metadata_quorum=[quorum.isolated_kraft],
         use_new_coordinator=[True, False]
     )
-    def test_transformations(self, connect_protocol, metadata_quorum, use_new_coordinator=False):
+    @matrix(
+        metadata_quorum=[quorum.isolated_kraft],
+        use_new_coordinator=[True],
+        group_protocol=consumer_group.all_group_protocols
+    )
```

Comment:
We still need the `connect_protocol` here:

```suggestion
    @matrix(
        connect_protocol=['sessioned', 'compatible', 'eager'],
        metadata_quorum=[quorum.isolated_kraft],
        use_new_coordinator=[True],
        group_protocol=consumer_group.all_group_protocols
    )
```